client/web/src/components/SaveToolbar.tsx

```diff
@@ -4,27 +4,43 @@ import CheckIcon from 'mdi-react/CheckIcon'
 import CloseIcon from 'mdi-react/CloseIcon'
 import * as React from 'react'
 
-interface Props {
+export interface Props {
```

Comment:
When it's exported, I'd make it a little more specific, because auto imports would be very bad otherwise

```suggestion
export interface SaveToolbarProps {
```